server/testutils/factories/cas1SpaceBooking.ts

```diff
@@ -107,5 +129,6 @@ export default Cas1SpaceBookingFactory.define(() => {
     otherBookingsInPremisesForCrn: cas1SpaceBookingDatesFactory.buildList(4),
     deliusEventNumber: String(faker.number.int()),
     nonArrival: undefined,
-  }
+    status: 'departed',
+  } as Cas1SpaceBooking
```

Comment:
Would casting `status` above as `Cas1SpaceBookingSummaryStatus` remove the need to type cast the whole faked booking?

Reply:
thanks, I'll try that.